docs/_newsfragments/2156.breakingchange.rst

```diff
@@ -0,0 +1,3 @@
+- **Deprecation**: Support for `testtools` in `falcon.testing` is now deprecated and will be removed in Falcon 5.0.
```

Comment:
Don't name the newsfragment `.breakingchange.`, because we are not making any actual breaking changes until 5.0.

Since we are adding a new way to configure the base test case, I suggest marketing this as new feature.